app/controllers/sign_in_tokens_controller.rb

```diff
@@ -109,7 +109,12 @@ def root_url_for(user)
 
   def school_user_start_url(user)
     if user.school_welcome_wizard&.complete?
-      school_home_path
+      if user.school&.preorder_information&.school_will_order_devices? &&
```

Comment:
Should `user.school&.preorder_information&.school_will_order_devices?` be incorporated into `chromebook_info_still_needed?`?

Reply:
i don't think so. I assume if the RB is ordering we will still ask them for chromebook details?